src/cmds/qstat.c

```diff
@@ -924,10 +961,14 @@ altdsp_statjob(struct batch_status *pstat, struct batch_status *prtheader, int a
 		else
 			timeval = usecput ? eltimecpu : eltimewal;
 
-
 		if (wide) {
+			if (strlen(pstat->name) > SIZEJOBID) {
```

Comment:
Should this be jname instead of pstat->name?

Reply:
jname hadn't been filled out yet.